webapp/app.py

```diff
@@ -304,6 +305,7 @@ def context():
         "CAPTCHA_TESTING_API_KEY": CAPTCHA_TESTING_API_KEY,
         "http_host": flask.request.host,
         "is_maintenance": strtobool(os.getenv("STORE_MAINTENANCE", "false")),
+        "current_user_country": get_user_country_by_ip(),
```

Comment:
It's not ideal to add this to the global context used for all pages if we only need it in one page. I don't know if this geolite2 DB looking takes much time, but it doesn't seem like the cheapest operation in the world.

Reply:
Also, why "current"? Does it change during the immediate session?

Reply:
We will be using this also for phone number prefix https://ubuntu.com/engage/apache-kafka-cli-cheat-sheet (phone number).
But yeah I have the same concern that you do. So I guess we can just pass this to every view that needs it

Reply:
Yeah that would be better.